plugins/filter_record_modifier/filter_modifier.c

```diff
@@ -293,6 +298,12 @@ static int cb_modifier_filter(const void *data, size_t bytes,
         /* grep keys */
         if (obj->type == MSGPACK_OBJECT_MAP) {
             map_num = obj->via.map.size;
+            /* allocate map_num + guard byte */
+            bool_map = flb_calloc(map_num+1, sizeof(bool_map_t));
```

Comment:
I think we should have a limit, what do you think ?

Reply:
I agree.

Do you have any idea about amount of limit ?
Is 4096 enough ? 65536 ? or configurable ? e.g. `map_size_limit 4096`

Reply:
I updated PR to add fixed limit 65535.